tools/clang/test/HLSL/rewriter/includes.hlsl

```diff
@@ -1,6 +1,6 @@
 #include "toinclude.hlsl"
 #include "..\toinclude2.hlsl"
-#include "\for_includes_test\toinclude3.hlsl"
+#include ".\for_includes_test\toinclude3.hlsl"
```

Comment:
Can you drop the `.\` entirely? The local directory should be on the include search path.
```suggestion
#include "for_includes_test\toinclude3.hlsl"
```

Reply:
Fixed.